django_netjsonconfig/base/admin.py

```diff
@@ -362,6 +365,62 @@ class AbstractTemplateAdmin(BaseConfigAdmin):
     class Media(BaseConfigAdmin.Media):
         js = BaseConfigAdmin.Media.js + [static('{0}js/{1}'.format(prefix, 'template_admin.js'))]
 
+    def delete_view(self, request, object_id, extra_context=None):
+        """
+        send unsubscription notifications to template designer
+        when deleted from the detail page.
+        """
+        if request.POST:
+            template = self.model.objects.get(pk=object_id)
+            subscriber_url = '{0}://{1}'.format(request.META.get('wsgi.url_scheme'),
+                                                request.get_host())
+            subscribe.delay(template.id, template.url, subscriber_url, subscribe=False)
+            self._delete_vpn(template.vpn_id)
+        return super(AbstractTemplateAdmin, self).delete_view(request, object_id, extra_context)
+
+    def subscribers(self, obj=None):
+        count = 0
+        if obj and obj.sharing == 'public' or obj.sharing == 'secret_key':
+            count = self.template_subscribe_model.objects.filter(subscribe=True, template=obj).count()
```

Comment:
same here, why can't we have a model method for this? We should then call the model method here.
This is how I'd do it, what do you think?